src/hooks/useFetch.js

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from 'react';
+
+const useFetch = url => {
+
+    const [loading, setLoading] = useState(true)
+    const [data, setData] = useState(null)
+
+    useEffect(() => {
+        async function fetchData() {
+          try{
+            const response = await fetch(url)
+            console.log(response)
+            setData(response.url)
+            setLoading(false)
+          } catch (e){
+            console.log(e)
+          }
+        }
+        fetchData()
```

Comment:
BEEEEEESTIAAAA! ME CAGAS!

Reply:
This code is so deep! It's wonderful and beautifully layered!